lib/src/scss-shared/_nx-colors.scss

```diff
@@ -101,12 +101,12 @@ $nx-border-row-disabled: 1px solid $nx-grey-1;
 $nx-border-row-dark: 1px solid $nx-grey-5;
 
 // Threat colors
-$nx-threat-color-unspecified: #d9dade;
-$nx-threat-color-none: $nx-light-blue;
-$nx-threat-color-low: $nx-dark-blue;
-$nx-threat-color-moderate: $nx-yellow;
-$nx-threat-color-severe: $nx-orange;
-$nx-threat-color-critical: $nx-dark-red;
+$nx-threat-color-unspecified: #d1d6f0;
+$nx-threat-color-none: #c5deff;
+$nx-threat-color-low: #045dd1;
+$nx-threat-color-moderate: #ffbf27;
+$nx-threat-color-severe: #ff8100;
+$nx-threat-color-critical: #cd0028;
 
 // Threat colors - light variant
 $nx-threat-color-none-light: #d9ecfb;
```

Comment:
Do these need to be updated too?  I think they're only used by the NxPolicyThreatSlider, so I guess if there's a ticket to update that component, these variables would be adjusted there.

Reply:
> Do these need to be updated too? I think they're only used by the NxPolicyThreatSlider, so I guess if there's a ticket to update that component, these variables would be adjusted there.

It would be nice to see them consolidated if possible, but I think that's a task for another ticket